backend/DB/migrations/20211017184350-create-intake-form-item.cjs

```diff
@@ -0,0 +1,81 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable(
+      "IntakeFormItems",
+      {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER,
+        },
+        type: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        createdAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        intakeFormId: {
+          type: Sequelize.INTEGER,
+          references: {
+            model: "IntakeForms",
+            key: "id",
+          },
+        },
+        fieldId: {
+          type: Sequelize.INTEGER,
+          references: {
+            model: "Fields",
+            key: "id",
+          },
+        },
+        intakeFormQuestionId: {
+          type: Sequelize.INTEGER,
+          references: {
+            model: "IntakeFormQuestions",
+            key: "id",
+          },
+        },
+        pageId: {
+          type: Sequelize.INTEGER,
+          references: {
+            model: "IntakeFormItems",
+            key: "id",
+          },
+        },
+        sectionId: {
```

Comment:
So sections are not stored in the database, they are just an integer on the intake items? How is nesting represented?